Providers/SelectedListManager.tscn

```diff
@@ -48,9 +49,20 @@ func delete_selected():
 			var index_to_remove = _find_entry_index(placemark)
 			if index_to_remove != -1:
 				selected_list.remove(index_to_remove)
+				close_details_panel()
 	
 func reveal_hidden():
 	emit_signal(\"set_revealed_hidden\")
+	
+func remove_placemark(placemark, data):
+	toggle_selection(placemark, data)
```

Comment:
What if this is called with a placemark that wasn't selected already. It'd be selected, doing the opposite of what it says

Reply:
Why not separate the toggle function into select and deselect functions